src/migrations/011_user-type.ts

```diff
@@ -0,0 +1,28 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({id: 11});
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
+  await knex('changelog').insert({
+    id: 11,
+    timestamp: Math.floor(Date.now()/1000)
+  });
+
+  await knex.raw(`ALTER TABLE oauth2_clients
+  DROP name`);
+  await knex.raw(`ALTER TABLE users
+  ADD type TINYINT UNSIGNED NOT NULL DEFAULT '1' COMMENT '1 = user, 2 = app'`);
```

Comment:
For example @evert this wont run in PostgreSQL because there is no `tinyint` type

Reply:
Yes, but if you run the file you generated (from posgres-schema/042-migration-from-mysql.sql) then all these lines should be skipped. It's a workaround until all these migrations are rewritten but it should work

Reply:
Right good point.